Covid/MainScreen/MainViewController.swift

```diff
@@ -66,6 +71,9 @@ final class MainViewController: UIViewController {
         super.viewDidAppear(animated)
         registerUser()
 
+//        after(.seconds(1)) { [weak self] in
```

Comment:
intermediate code == by uncommenting this code, verification FaceID flow can be tested

Reply:
este, nepovolime ten faceid nech to aj chalani vedia otestit?

Reply:
rozmyslal som nad tym. eg pridat moznost do settingsov appky. tzn jednoduchy switch, ktory by vyhadzoval flow na verifikaciu pri starte appky, teda za predpokladu, ze faceID template je po ruke == uzivatel presiel faceID registraciou

Reply:
a sak nekomplikujme si to, nech aj ti backendaci nieco porobia